test/integration/integration_admin_test.cc

```diff
@@ -289,6 +291,12 @@ TEST_P(IntegrationAdminTest, Admin) {
   EXPECT_THAT(response->body(), testing::HasSubstr("added_via_api"));
   EXPECT_EQ("text/plain; charset=UTF-8", ContentType(response));
 
+  response = IntegrationUtil::makeSingleRequest(lookupPort("admin"), "GET", "/clusters?format=json",
+                                                "", downstreamProtocol(), version_);
+  EXPECT_TRUE(response->complete());
+  EXPECT_EQ("200", response->headers().Status()->value().getStringView());
+  EXPECT_EQ("application/json", ContentType(response));
```

Comment:
can we validate here that json is parsable? Ideally we could validate with some actual JSON but atleast we should validate that the output is parsable.

Reply:
done